plugins/Kaleidoscope-MouseKeys/src/kaleidoscope/plugin/MouseKeys.cpp

```diff
@@ -52,7 +52,16 @@ uint16_t MouseKeys::wheelDelay = 50;
 // Configuration functions
 
 void MouseKeys::setWarpGridSize(uint8_t grid_size) {
+  // Sanity check the grid size here, so we do not need to do that at the call
+  // sites. When given an invalid grid size, just don't touch the setting.
+  if (grid_size < 2 || grid_size > 3) return;
```

Comment:
I missed this line on my first review.

How do we handle the error reporting in-protocol?